pkg/operator/operator.go

```diff
@@ -445,19 +448,36 @@ func (op *Reporting) Run(ctx context.Context) error {
 
 	// check if the PrestoUseTLS flag is set to true
 	if op.cfg.PrestoUseTLS {
-		cert, err := ioutil.ReadFile(op.cfg.PrestoCAFile)
+		prestoTLSConfig := &tls.Config{}
```

Comment:
Why not just initialize this at the same time as you set the RootCAs? Eg remove this line, change https://github.com/operator-framework/operator-metering/pull/739/files#diff-554de5523753fda8c93e7008c9bd947fR461 to use `:=`

Reply:
yeah, my mistake I should've changed that after making the previous changes.